diff --git a/sys/dev/usb/uhidev.c b/sys/dev/usb/uhidev.c
--- a/sys/dev/usb/uhidev.c
+++ b/sys/dev/usb/uhidev.c
@@ -152,6 +152,15 @@ uhidev_attach(device_t parent, device_t self, void *aux)
 	if (!pmf_device_register(self, NULL, NULL))
 		aprint_error_dev(self, "couldn't establish power handler\n");
 
+	if (uiaa->uiaa_vendor == USB_VENDOR_WACOM) {
+		if (uiaa->uiaa_product == USB_PRODUCT_WACOM_XD0912U) {
+		/*
+		 * Wacom Intuos2 (XD-0912-U) requires longer idle time to
+		 * initialize the device with 0x0202.
+		 */
+			DELAY(500000);
+		}
+	}
 	(void)usbd_set_idle(iface, 0, 0);
 
 	if ((usbd_get_quirks(sc->sc_udev)->uq_flags & UQ_NO_SET_PROTO) == 0)
@@ -202,25 +211,25 @@ uhidev_attach(device_t parent, device_t self, void *aux)
 	/* XXX need to extend this */
 	descptr = NULL;
 	if (uiaa->uiaa_vendor == USB_VENDOR_WACOM) {
-		static uByte reportbuf[] = {2, 2, 2};
+		static uByte reportbuf[3];
 
 		/* The report descriptor for the Wacom Graphire is broken. */
 		switch (uiaa->uiaa_product) {
-		case USB_PRODUCT_WACOM_GRAPHIRE:
-		case USB_PRODUCT_WACOM_GRAPHIRE2:
 		case USB_PRODUCT_WACOM_GRAPHIRE3_4X5:
 		case USB_PRODUCT_WACOM_GRAPHIRE3_6X8:
-		case USB_PRODUCT_WACOM_GRAPHIRE4_4X5: /* The 6x8 too? */
-			/*
-			 * The Graphire3 needs 0x0202 to be written to
-			 * feature report ID 2 before it'll start
-			 * returning digitizer data.
-			 */
+		case USB_PRODUCT_WACOM_GRAPHIRE4_4X5:
+		case USB_PRODUCT_WACOM_GRAPHIRE:
+		case USB_PRODUCT_WACOM_GRAPHIRE2:
+		case USB_PRODUCT_WACOM_XD0912U:
+		case USB_PRODUCT_WACOM_CTL470K:
+		case USB_PRODUCT_WACOM_CTE650:
+		case USB_PRODUCT_WACOM_CTH670K:
+		case USB_PRODUCT_WACOM_CTH480:
+		case USB_PRODUCT_WACOM_CTH690K0:
+			reportbuf[0] = 0x02;
+			reportbuf[1] = 0x02;
 			usbd_set_report(uiaa->uiaa_iface, UHID_FEATURE_REPORT, 2,
-			    &reportbuf, sizeof(reportbuf));
-
-			size = sizeof(uhid_graphire3_4x5_report_descr);
-			descptr = uhid_graphire3_4x5_report_descr;
+			    &reportbuf, 2);
 			break;
 		default:
 			/* Keep descriptor */
diff --git a/sys/dev/usb/usbdevs b/sys/dev/usb/usbdevs
--- a/sys/dev/usb/usbdevs
+++ b/sys/dev/usb/usbdevs
@@ -3389,8 +3389,14 @@ product WACOM GRAPHIRE2		0x0011	Graphire2 ET-0405A-U
 product WACOM GRAPHIRE3_4X5	0x0013	Graphire3 4x5
 product WACOM GRAPHIRE3_6X8	0x0014	Graphire3 6x8
 product WACOM GRAPHIRE4_4X5	0x0015	Graphire4 4x5
+product WACOM CTE650		0x0018	Bamboo Fan medium CTE-650
 product WACOM INTUOSA5		0x0021	Intuos A5
 product WACOM GD0912U		0x0022	Intuos 9x12 Graphics Tablet
+product WACOM XD0912U		0x0043	Intuos2 A4 i-920 XD-0912-U
+product WACOM CTL470K		0x00dd	Bamboo Pen CTH-470/K
+product WACOM CTH670K		0x00df	Bamboo Pen and Touch medium CTH-670/K
+product WACOM CTH480		0x0302	Intuos Pen and Touch small CTH-480
+product WACOM CTH690K0		0x033e	Intuos Art CTH-690/K0
 
 /* Weltrend Semiconductor */
 product WELTREND HID		0x2201	HID Device
diff --git a/sys/dev/usb/usbdevs.h b/sys/dev/usb/usbdevs.h
--- a/sys/dev/usb/usbdevs.h
+++ b/sys/dev/usb/usbdevs.h
@@ -1,4 +1,4 @@
-/*	$NetBSD: usbdevs.h,v 1.727 2017/05/30 20:16:08 jnemeth Exp $	*/
+/*	$NetBSD$	*/
 
 /*
  * THIS FILE AUTOMATICALLY GENERATED.  DO NOT EDIT.
@@ -3396,8 +3396,14 @@
 #define	USB_PRODUCT_WACOM_GRAPHIRE3_4X5	0x0013		/* Graphire3 4x5 */
 #define	USB_PRODUCT_WACOM_GRAPHIRE3_6X8	0x0014		/* Graphire3 6x8 */
 #define	USB_PRODUCT_WACOM_GRAPHIRE4_4X5	0x0015		/* Graphire4 4x5 */
+#define	USB_PRODUCT_WACOM_CTE650	0x0018		/* Bamboo Fan medium CTE-650 */
 #define	USB_PRODUCT_WACOM_INTUOSA5	0x0021		/* Intuos A5 */
 #define	USB_PRODUCT_WACOM_GD0912U	0x0022		/* Intuos 9x12 Graphics Tablet */
+#define	USB_PRODUCT_WACOM_XD0912U	0x0043		/* Intuos2 A4 i-920 XD-0912-U */
+#define	USB_PRODUCT_WACOM_CTL470K	0x00dd		/* Bamboo Pen CTH-470/K */
+#define	USB_PRODUCT_WACOM_CTH670K	0x00df		/* Bamboo Pen and Touch medium CTH-670/K */
+#define	USB_PRODUCT_WACOM_CTH480	0x0302		/* Intuos Pen and Touch small CTH-480 */
+#define	USB_PRODUCT_WACOM_CTH690K0	0x033e		/* Intuos Art CTH-690/K0 */
 
 /* Weltrend Semiconductor */
 #define	USB_PRODUCT_WELTREND_HID	0x2201		/* HID Device */
diff --git a/sys/dev/usb/usbdevs_data.h b/sys/dev/usb/usbdevs_data.h
--- a/sys/dev/usb/usbdevs_data.h
+++ b/sys/dev/usb/usbdevs_data.h
@@ -1,4 +1,4 @@
-/*	$NetBSD: usbdevs_data.h,v 1.728 2017/05/30 20:16:08 jnemeth Exp $	*/
+/*	$NetBSD$	*/
 
 /*
  * THIS FILE AUTOMATICALLY GENERATED.  DO NOT EDIT.
@@ -4592,28 +4592,40 @@ static const uint16_t usb_products[] = {
 	    19172, 19186, 0,
 	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_GRAPHIRE4_4X5, 
 	    19190, 19182, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_CTE650, 
+	    19200, 19207, 19211, 19218, 0,
 	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_INTUOSA5, 
-	    19200, 19207, 0,
+	    19226, 19233, 0,
 	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_GD0912U, 
-	    19200, 19210, 1886, 19135, 0,
+	    19226, 19236, 1886, 19135, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_XD0912U, 
+	    19241, 18281, 19249, 19255, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_CTL470K, 
+	    19200, 3507, 19265, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_CTH670K, 
+	    19200, 3507, 19275, 5592, 19211, 19279, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_CTH480, 
+	    19226, 3507, 19275, 5592, 19289, 19295, 0,
+	    USB_VENDOR_WACOM, USB_PRODUCT_WACOM_CTH690K0, 
+	    19226, 19303, 19307, 0,
 	    USB_VENDOR_WELTREND, USB_PRODUCT_WELTREND_HID, 
-	    19215, 7711, 0,
+	    19318, 7711, 0,
 	    USB_VENDOR_WESTERN, USB_PRODUCT_WESTERN_EXTHDD, 
-	    19219, 19228, 0,
+	    19322, 19331, 0,
 	    USB_VENDOR_WINCHIPHEAD, USB_PRODUCT_WINCHIPHEAD_CH341SER, 
-	    19232, 16941, 1786, 0,
+	    19335, 16941, 1786, 0,
 	    USB_VENDOR_WINCHIPHEAD2, USB_PRODUCT_WINCHIPHEAD2_CH341, 
-	    19244, 16941, 1786, 0,
+	    19347, 16941, 1786, 0,
 	    USB_VENDOR_WINCHIPHEAD2, USB_PRODUCT_WINCHIPHEAD2_CH341_2, 
-	    19244, 16941, 1786, 0,
+	    19347, 16941, 1786, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_WNC0600, 
-	    19250, 0,
+	    19353, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_UR045G, 
 	    5210, 4743, 5218, 5183, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_UR055G, 
-	    19262, 0,
+	    19365, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_O8494, 
-	    19269, 19277, 0,
+	    19372, 19380, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_AR5523_1, 
 	    6807, 0,
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_AR5523_1_NF, 
@@ -4623,7 +4635,7 @@ static const uint16_t usb_products[] = {
 	    USB_VENDOR_WISTRONNEWEB, USB_PRODUCT_WISTRONNEWEB_AR5523_2_NF, 
 	    6807, 0,
 	    USB_VENDOR_XIRLINK, USB_PRODUCT_XIRLINK_IMAGING, 
-	    19285, 19293, 0,
+	    19388, 19396, 0,
 	    USB_VENDOR_XIRLINK, USB_PRODUCT_XIRLINK_PCCAM, 
 	    672, 5667, 2901, 0,
 	    USB_VENDOR_CONEXANT, USB_PRODUCT_CONEXANT_MODEM_1, 
@@ -4633,53 +4645,53 @@ static const uint16_t usb_products[] = {
 	    USB_VENDOR_CONEXANT, USB_PRODUCT_CONEXANT_PRISM_GT_2, 
 	    5210, 4743, 5218, 5183, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_UX256, 
-	    19300, 16449, 16440, 0,
+	    19403, 16449, 16440, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MU1000, 
-	    19306, 16449, 16484, 0,
+	    19409, 16449, 16484, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MU2000, 
-	    19313, 16449, 16484, 0,
+	    19416, 16449, 16484, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MU500, 
-	    19320, 16449, 16484, 0,
+	    19423, 16449, 16484, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_UW500, 
-	    19326, 4743, 7058, 16440, 0,
+	    19429, 4743, 7058, 16440, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MOTIF6, 
-	    19332, 16449, 16484, 19339, 0,
+	    19435, 16449, 16484, 19442, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MOTIF7, 
-	    19351, 16449, 16484, 19339, 0,
+	    19454, 16449, 16484, 19442, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_MOTIF8, 
-	    19358, 16449, 16484, 19339, 0,
+	    19461, 16449, 16484, 19442, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_UX96, 
-	    19365, 16449, 16440, 0,
+	    19468, 16449, 16440, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_UX16, 
-	    19370, 16449, 16440, 0,
+	    19473, 16449, 16440, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_S08, 
-	    19375, 16449, 5759, 0,
+	    19478, 16449, 5759, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_CLP150, 
-	    19379, 14275, 19387, 0,
+	    19482, 14275, 19490, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_CLP170, 
-	    19393, 14275, 19387, 0,
+	    19496, 14275, 19490, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_RPU200, 
-	    19401, 0,
+	    19504, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_RTA54I, 
-	    19409, 19420, 19427, 18412, 0,
+	    19512, 19523, 19530, 18412, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_RTW65B, 
-	    19409, 19442, 18402, 3677, 18412, 0,
+	    19512, 19545, 18402, 3677, 18412, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_RTW65I, 
-	    19409, 19449, 19427, 3677, 18412, 0,
+	    19512, 19552, 19530, 3677, 18412, 0,
 	    USB_VENDOR_YAMAHA, USB_PRODUCT_YAMAHA_RTA55I, 
-	    19409, 19456, 18402, 19463, 18412, 0,
+	    19512, 19559, 18402, 19566, 18412, 0,
 	    USB_VENDOR_YANO, USB_PRODUCT_YANO_U640MO, 
-	    19468, 0,
+	    19571, 0,
 	    USB_VENDOR_YEDATA, USB_PRODUCT_YEDATA_FLASHBUSTERU, 
-	    19478, 0,
+	    19581, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_M4Y750, 
-	    19492, 0,
+	    19595, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_725, 
-	    19500, 5374, 5183, 0,
+	    19603, 5374, 5183, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_XI735, 
-	    19508, 0,
+	    19611, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_MD40900, 
-	    19515, 0,
+	    19618, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_XG703A, 
 	    5210, 4743, 5218, 5183, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_ZD1211, 
@@ -4693,15 +4705,15 @@ static const uint16_t usb_products[] = {
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_RT2870_1, 
 	    4969, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_UB81, 
-	    19523, 0,
+	    19626, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_RT2870_2, 
 	    4969, 0,
 	    USB_VENDOR_ZCOM, USB_PRODUCT_ZCOM_UB82, 
-	    19528, 0,
+	    19631, 0,
 	    USB_VENDOR_ZEEVO, USB_PRODUCT_ZEEVO_BLUETOOTH, 
-	    19533, 4747, 4743, 4757, 0,
+	    19636, 4747, 4743, 4757, 0,
 	    USB_VENDOR_ZINWELL, USB_PRODUCT_ZINWELL_ZWXG261, 
-	    19540, 0,
+	    19643, 0,
 	    USB_VENDOR_ZINWELL, USB_PRODUCT_ZINWELL_RT2870_1, 
 	    4969, 0,
 	    USB_VENDOR_ZINWELL, USB_PRODUCT_ZINWELL_RT2870_2, 
@@ -4713,73 +4725,73 @@ static const uint16_t usb_products[] = {
 	    USB_VENDOR_ZINWELL, USB_PRODUCT_ZINWELL_RT3070, 
 	    4976, 0,
 	    USB_VENDOR_ZOOM, USB_PRODUCT_ZOOM_2986L, 
-	    19549, 19555, 5834, 0,
+	    19652, 19658, 5834, 0,
 	    USB_VENDOR_ZOOM, USB_PRODUCT_ZOOM_3095, 
-	    19559, 4743, 19555, 5834, 0,
+	    19662, 4743, 19658, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MF622, 
-	    19564, 5834, 0,
+	    19667, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MF628, 
-	    19570, 5834, 0,
+	    19673, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MF626, 
-	    19576, 5834, 0,
+	    19679, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MF820D_INSTALLER, 
-	    19582, 11138, 0,
+	    19685, 11138, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MF820D, 
-	    19582, 5834, 0,
+	    19685, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_INSTALLER, 
-	    19589, 11138, 0,
+	    19692, 11138, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_MC2718, 
-	    19594, 5834, 0,
+	    19697, 5834, 0,
 	    USB_VENDOR_ZTE, USB_PRODUCT_ZTE_AC8700, 
-	    5662, 19601, 4743, 5834, 0,
+	    5662, 19704, 4743, 5834, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ZD1201, 
-	    19608, 0,
+	    19711, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ZD1211, 
-	    18362, 5183, 19615, 0,
+	    18362, 5183, 19718, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ZD1211B, 
 	    5258, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ZD1221, 
-	    19619, 0,
+	    19722, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ALL0298, 
 	    18894, 0,
 	    USB_VENDOR_ZYDAS, USB_PRODUCT_ZYDAS_ZD1211B_2, 
 	    5258, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_OMNI56K, 
-	    19626, 18309, 10766, 0,
+	    19729, 18309, 10766, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_980N, 
 	    16094, 7725, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_ZYAIRG220, 
-	    18542, 19631, 0,
+	    18542, 19734, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_G200V2, 
-	    19637, 7346, 0,
+	    19740, 7346, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_AG225H, 
-	    19643, 0,
+	    19746, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_M202, 
-	    19651, 0,
+	    19754, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_G270S, 
-	    19657, 0,
+	    19760, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_G220V2, 
-	    19631, 7346, 0,
+	    19734, 7346, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_G202, 
-	    19664, 0,
+	    19767, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_RT2573, 
 	    5140, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_RT2870_1, 
 	    4969, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_NWD271N, 
-	    19670, 0,
+	    19773, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_NWD211AN, 
-	    19679, 0,
+	    19782, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_RT2870_2, 
 	    4969, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_NWD2105, 
-	    19689, 0,
+	    19792, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_RTL8192CU, 
 	    5090, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_RT3070, 
 	    4976, 0,
 	    USB_VENDOR_ZYXEL, USB_PRODUCT_ZYXEL_PRESTIGE, 
-	    19697, 0,
+	    19800, 0,
 };
 static const char usb_words[] = { "." 
 	    "Unknown\0" /* 4 refs @ 1 */
@@ -5241,7 +5253,7 @@ static const char usb_words[] = { "."
 	    "Z-Com\0" /* 1 refs @ 3485 */
 	    "Atheros\0" /* 2 refs @ 3491 */
 	    "Tangtop\0" /* 1 refs @ 3499 */
-	    "Pen\0" /* 4 refs @ 3507 */
+	    "Pen\0" /* 7 refs @ 3507 */
 	    "Drive\0" /* 11 refs @ 3511 */
 	    "C-Media\0" /* 1 refs @ 3517 */
 	    "Conceptronic\0" /* 2 refs @ 3525 */
@@ -5525,7 +5537,7 @@ static const char usb_words[] = { "."
 	    "SnapScan\0" /* 10 refs @ 5571 */
 	    "1212U\0" /* 2 refs @ 5580 */
 	    "1236U\0" /* 2 refs @ 5586 */
-	    "Touch\0" /* 7 refs @ 5592 */
+	    "Touch\0" /* 9 refs @ 5592 */
 	    "e40\0" /* 1 refs @ 5598 */
 	    "e50\0" /* 1 refs @ 5602 */
 	    "e20\0" /* 1 refs @ 5606 */
@@ -7204,7 +7216,7 @@ static const char usb_words[] = { "."
 	    "U\0" /* 1 refs @ 18265 */
 	    "Ir-Trinity\0" /* 1 refs @ 18267 */
 	    "A3\0" /* 1 refs @ 18278 */
-	    "A4\0" /* 1 refs @ 18281 */
+	    "A4\0" /* 2 refs @ 18281 */
 	    "Type\0" /* 3 refs @ 18284 */
 	    "SR9600\0" /* 1 refs @ 18289 */
 	    "QF9700\0" /* 1 refs @ 18296 */
@@ -7331,77 +7343,91 @@ static const char usb_words[] = { "."
 	    "4x5\0" /* 2 refs @ 19182 */
 	    "6x8\0" /* 1 refs @ 19186 */
 	    "Graphire4\0" /* 1 refs @ 19190 */
-	    "Intuos\0" /* 2 refs @ 19200 */
-	    "A5\0" /* 1 refs @ 19207 */
-	    "9x12\0" /* 1 refs @ 19210 */
-	    "HID\0" /* 1 refs @ 19215 */
-	    "External\0" /* 1 refs @ 19219 */
-	    "HDD\0" /* 1 refs @ 19228 */
-	    "CH341/CH340\0" /* 1 refs @ 19232 */
-	    "CH341\0" /* 2 refs @ 19244 */
-	    "WNC-0600USB\0" /* 1 refs @ 19250 */
-	    "UR055G\0" /* 1 refs @ 19262 */
-	    "ORiNOCO\0" /* 1 refs @ 19269 */
-	    "802.11n\0" /* 1 refs @ 19277 */
-	    "IMAGING\0" /* 1 refs @ 19285 */
-	    "DEVICE\0" /* 1 refs @ 19293 */
-	    "UX256\0" /* 1 refs @ 19300 */
-	    "MU1000\0" /* 1 refs @ 19306 */
-	    "MU2000\0" /* 1 refs @ 19313 */
-	    "MU500\0" /* 1 refs @ 19320 */
-	    "UW500\0" /* 1 refs @ 19326 */
-	    "MOTIF6\0" /* 1 refs @ 19332 */
-	    "Workstation\0" /* 3 refs @ 19339 */
-	    "MOTIF7\0" /* 1 refs @ 19351 */
-	    "MOTIF8\0" /* 1 refs @ 19358 */
-	    "UX96\0" /* 1 refs @ 19365 */
-	    "UX16\0" /* 1 refs @ 19370 */
-	    "S08\0" /* 1 refs @ 19375 */
-	    "CLP-150\0" /* 1 refs @ 19379 */
-	    "piano\0" /* 2 refs @ 19387 */
-	    "CLP-170\0" /* 1 refs @ 19393 */
-	    "RP-U200\0" /* 1 refs @ 19401 */
-	    "NetVolante\0" /* 4 refs @ 19409 */
-	    "RTA54i\0" /* 1 refs @ 19420 */
-	    "Broadband&ISDN\0" /* 2 refs @ 19427 */
-	    "RTW65b\0" /* 1 refs @ 19442 */
-	    "RTW65i\0" /* 1 refs @ 19449 */
-	    "RTA55i\0" /* 1 refs @ 19456 */
-	    "VoIP\0" /* 1 refs @ 19463 */
-	    "U640MO-03\0" /* 1 refs @ 19468 */
-	    "Flashbuster-U\0" /* 1 refs @ 19478 */
-	    "M4Y-750\0" /* 1 refs @ 19492 */
-	    "725/726\0" /* 1 refs @ 19500 */
-	    "XI-735\0" /* 1 refs @ 19508 */
-	    "MD40900\0" /* 1 refs @ 19515 */
-	    "UB81\0" /* 1 refs @ 19523 */
-	    "UB82\0" /* 1 refs @ 19528 */
-	    "BT-500\0" /* 1 refs @ 19533 */
-	    "ZWX-G261\0" /* 1 refs @ 19540 */
-	    "2986L\0" /* 1 refs @ 19549 */
-	    "Fax\0" /* 2 refs @ 19555 */
-	    "3095\0" /* 1 refs @ 19559 */
-	    "MF622\0" /* 1 refs @ 19564 */
-	    "MF628\0" /* 1 refs @ 19570 */
-	    "MF626\0" /* 1 refs @ 19576 */
-	    "MF820D\0" /* 2 refs @ 19582 */
-	    "UMTS\0" /* 1 refs @ 19589 */
-	    "MC2718\0" /* 1 refs @ 19594 */
-	    "1xEVDO\0" /* 1 refs @ 19601 */
-	    "ZD1201\0" /* 1 refs @ 19608 */
-	    "abg\0" /* 1 refs @ 19615 */
-	    "ZD1221\0" /* 1 refs @ 19619 */
-	    "Omni\0" /* 1 refs @ 19626 */
-	    "G-220\0" /* 2 refs @ 19631 */
-	    "G-200\0" /* 1 refs @ 19637 */
-	    "AG-225H\0" /* 1 refs @ 19643 */
-	    "M-202\0" /* 1 refs @ 19651 */
-	    "G-270S\0" /* 1 refs @ 19657 */
-	    "G-202\0" /* 1 refs @ 19664 */
-	    "NWD-271N\0" /* 1 refs @ 19670 */
-	    "NWD-211AN\0" /* 1 refs @ 19679 */
-	    "NWD2105\0" /* 1 refs @ 19689 */
-	    "Prestige\0" /* 1 refs @ 19697 */
+	    "Bamboo\0" /* 3 refs @ 19200 */
+	    "Fan\0" /* 1 refs @ 19207 */
+	    "medium\0" /* 2 refs @ 19211 */
+	    "CTE-650\0" /* 1 refs @ 19218 */
+	    "Intuos\0" /* 4 refs @ 19226 */
+	    "A5\0" /* 1 refs @ 19233 */
+	    "9x12\0" /* 1 refs @ 19236 */
+	    "Intuos2\0" /* 1 refs @ 19241 */
+	    "i-920\0" /* 1 refs @ 19249 */
+	    "XD-0912-U\0" /* 1 refs @ 19255 */
+	    "CTH-470/K\0" /* 1 refs @ 19265 */
+	    "and\0" /* 2 refs @ 19275 */
+	    "CTH-670/K\0" /* 1 refs @ 19279 */
+	    "small\0" /* 1 refs @ 19289 */
+	    "CTH-480\0" /* 1 refs @ 19295 */
+	    "Art\0" /* 1 refs @ 19303 */
+	    "CTH-690/K0\0" /* 1 refs @ 19307 */
+	    "HID\0" /* 1 refs @ 19318 */
+	    "External\0" /* 1 refs @ 19322 */
+	    "HDD\0" /* 1 refs @ 19331 */
+	    "CH341/CH340\0" /* 1 refs @ 19335 */
+	    "CH341\0" /* 2 refs @ 19347 */
+	    "WNC-0600USB\0" /* 1 refs @ 19353 */
+	    "UR055G\0" /* 1 refs @ 19365 */
+	    "ORiNOCO\0" /* 1 refs @ 19372 */
+	    "802.11n\0" /* 1 refs @ 19380 */
+	    "IMAGING\0" /* 1 refs @ 19388 */
+	    "DEVICE\0" /* 1 refs @ 19396 */
+	    "UX256\0" /* 1 refs @ 19403 */
+	    "MU1000\0" /* 1 refs @ 19409 */
+	    "MU2000\0" /* 1 refs @ 19416 */
+	    "MU500\0" /* 1 refs @ 19423 */
+	    "UW500\0" /* 1 refs @ 19429 */
+	    "MOTIF6\0" /* 1 refs @ 19435 */
+	    "Workstation\0" /* 3 refs @ 19442 */
+	    "MOTIF7\0" /* 1 refs @ 19454 */
+	    "MOTIF8\0" /* 1 refs @ 19461 */
+	    "UX96\0" /* 1 refs @ 19468 */
+	    "UX16\0" /* 1 refs @ 19473 */
+	    "S08\0" /* 1 refs @ 19478 */
+	    "CLP-150\0" /* 1 refs @ 19482 */
+	    "piano\0" /* 2 refs @ 19490 */
+	    "CLP-170\0" /* 1 refs @ 19496 */
+	    "RP-U200\0" /* 1 refs @ 19504 */
+	    "NetVolante\0" /* 4 refs @ 19512 */
+	    "RTA54i\0" /* 1 refs @ 19523 */
+	    "Broadband&ISDN\0" /* 2 refs @ 19530 */
+	    "RTW65b\0" /* 1 refs @ 19545 */
+	    "RTW65i\0" /* 1 refs @ 19552 */
+	    "RTA55i\0" /* 1 refs @ 19559 */
+	    "VoIP\0" /* 1 refs @ 19566 */
+	    "U640MO-03\0" /* 1 refs @ 19571 */
+	    "Flashbuster-U\0" /* 1 refs @ 19581 */
+	    "M4Y-750\0" /* 1 refs @ 19595 */
+	    "725/726\0" /* 1 refs @ 19603 */
+	    "XI-735\0" /* 1 refs @ 19611 */
+	    "MD40900\0" /* 1 refs @ 19618 */
+	    "UB81\0" /* 1 refs @ 19626 */
+	    "UB82\0" /* 1 refs @ 19631 */
+	    "BT-500\0" /* 1 refs @ 19636 */
+	    "ZWX-G261\0" /* 1 refs @ 19643 */
+	    "2986L\0" /* 1 refs @ 19652 */
+	    "Fax\0" /* 2 refs @ 19658 */
+	    "3095\0" /* 1 refs @ 19662 */
+	    "MF622\0" /* 1 refs @ 19667 */
+	    "MF628\0" /* 1 refs @ 19673 */
+	    "MF626\0" /* 1 refs @ 19679 */
+	    "MF820D\0" /* 2 refs @ 19685 */
+	    "UMTS\0" /* 1 refs @ 19692 */
+	    "MC2718\0" /* 1 refs @ 19697 */
+	    "1xEVDO\0" /* 1 refs @ 19704 */
+	    "ZD1201\0" /* 1 refs @ 19711 */
+	    "abg\0" /* 1 refs @ 19718 */
+	    "ZD1221\0" /* 1 refs @ 19722 */
+	    "Omni\0" /* 1 refs @ 19729 */
+	    "G-220\0" /* 2 refs @ 19734 */
+	    "G-200\0" /* 1 refs @ 19740 */
+	    "AG-225H\0" /* 1 refs @ 19746 */
+	    "M-202\0" /* 1 refs @ 19754 */
+	    "G-270S\0" /* 1 refs @ 19760 */
+	    "G-202\0" /* 1 refs @ 19767 */
+	    "NWD-271N\0" /* 1 refs @ 19773 */
+	    "NWD-211AN\0" /* 1 refs @ 19782 */
+	    "NWD2105\0" /* 1 refs @ 19792 */
+	    "Prestige\0" /* 1 refs @ 19800 */
 };
-const int usb_nwords = 2620;
+const int usb_nwords = 2634;
 
